tools/csv-merger/csv-merger.go

```diff
@@ -437,6 +450,13 @@ func generateUnifiedCSV() *csvv1.ClusterServiceVersion {
 			}
 		}
 	}
+	// whitelisting APIs
+	for _, definition := range ocsCSV.Spec.CustomResourceDefinitions.Owned {
+		if !contains(nonInternalAPIs, definition.Name) {
+			ocsCSV.Annotations["operators.operatorframework.io/internal-objects"] = ocsCSV.Annotations["operators.operatorframework.io/internal-objects"] + "," + "\"" + definition.Name + "\""
+		}
+	}
+	ocsCSV.Annotations["operators.operatorframework.io/internal-objects"] = "[" + strings.Replace(ocsCSV.Annotations["operators.operatorframework.io/internal-objects"], ",", "", 1) + "]"
```

Comment:
Instead of doing a replace, use the index from the range in the for loop above to see if you're at index zero, and if so add `[` instead of `,`. Then here you can just add the final `]`